### PR TITLE
Tickets/dm 23842

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: python/lsst/ts/Dome/dome_csc.py [hunk 1]
```diff
@@ -21,47 +21,65 @@ def __init__(
...
             config_dir=config_dir,
             initial_state=initial_state,
             simulation_mode=simulation_mode,
         )
 
-    async def do_crawlAz(self, data):
+    async def do_moveAz(self, data):
         """
-        Crawl AZ
+        Move AZ
```
Comment (r-owen) [inline python/lsst/ts/Dome/dome_csc.py]: Really pick point, but please put the brief description on the same line as the opening `"""`. In general we follow DM's convention for Python doc strings: https://developer.lsst.io/python/numpydoc.html (and DM basically follows numpy's convention).

File: python/lsst/ts/Dome/dome_csc.py [hunk 2]
```diff
@@ -108,44 +126,88 @@ def __init__(
...
         """
-        Set Temperature
+        Internal Fans command
         """
         self.assert_enabled()
         raise salobj.ExpectedError("Not implemented")
 
+    async def inflate(self, data):
+        """
+        Internal Inflate command
```
Comment (r-owen) [inline python/lsst/ts/Dome/dome_csc.py]: This is not very informative. What does "Internal" mean in this context? What does "Inflate" mean?

File: tests/test_dome_csc.py [hunk 3]
```diff
@@ -13,19 +13,53 @@ def basic_make_csc(self, initial_state, config_dir, simulation_mode):
...
+                    "closeShutter",
+                    "stopShutter",
+                    "park",
+                    "setTemperature",
+                    "config",
+                    "fans",
+                    "inflate",
+                    "status",
+                )
+            )
```
Comment (r-owen) [inline tests/test_dome_csc.py]: Thank you

File: tests/test_dome_csc.py [hunk 4]
```diff
@@ -37,23 +71,29 @@ def basic_make_csc(self, initial_state, config_dir, simulation_mode):
...
         pass
 
-    async def test_do_setTemperature(self):
+    async def test_fans(self):
+        pass
+
+    async def test_inflate(self):
+        pass
+
+    async def test_status(self):
```
Comment (r-owen) [inline tests/test_dome_csc.py]: I concerned that all these null tests will make it difficult to figure out which tests are not yet implemented (especially once you have implemented most of them). Consider replacing `pass` with `raise unittest.SkipTest("Not implemented")` or something similar.

File: python/lsst/ts/Dome/dome_csc.py (hunk 1 above)
Comment (wvreeven) [reply python/lsst/ts/Dome/dome_csc.py]: Done

File: python/lsst/ts/Dome/dome_csc.py (hunk 2 above)
Comment (wvreeven) [reply python/lsst/ts/Dome/dome_csc.py]: I added more descriptive inline comments for these "internal" functions.

File: tests/test_dome_csc.py (hunk 4 above)
Comment (wvreeven) [reply tests/test_dome_csc.py]: Done

**Pull request merged**